src/plugins/recordTypes/media/fields.js

```diff
@@ -0,0 +1,91 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    TextInput,
+    OptionPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_INT,
+  } = configContext.dataTypes;
+
+  return {
+    document: {
+      'ns2:media_botgarden': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/media/local/botgarden',
+          },
+        },
+        handwritten: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.media_botgarden.handwritten.name',
+                defaultMessage: 'Handwritten label',
+              },
+            }),
+            view: {
+              type: OptionPickerInput,
+              props: {
+                source: 'booleans',
+              },
+            },
+          },
+        },
+        postToPublic: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.media_botgarden.postToPublic.name',
+                defaultMessage: 'Post to public',
+              },
+            }),
+            view: {
+              type: OptionPickerInput,
+              props: {
+                source: 'booleans',
```

Comment:
Can't use the `booleans` list because the values don't match. I added a new list to cspace-ui called `yesNoValues` that you can use here (need to update your cspace-ui dependency to 1.0.0-rc.19).

(You can't use `yesNoValues` for handwritten, because it has that `Unknown` value, so it still needs a new list).